src/CI/CI.csproj

```diff
@@ -39,7 +39,7 @@
   </PropertyGroup>
   <ItemGroup>
     <None Include="App.config" />
-    <None Include="Axe.Windows.nuspec">
+    <None Include="Microsoft.Axe.Windows.nuspec">
```

Comment:
Do we _need_ to rename the file?

Reply:
Discussed offline, we will leave as-is - trade-off between unhelpful prefixes for navigation and name of file matching name of package